www/app/_components/common/Container.tsx

```diff
@@ -12,6 +13,15 @@ export default function Container({
 }) {
   return (
     <Body background={background}>
+      <Script src="https://www.googletagmanager.com/gtag/js?id=G-KE7812B4ZF" />
+      <Script id="google-analytics">
+        {`
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+        gtag('config', 'G-KE7812B4ZF');
```

Comment:
this should be an env var, for posterity, also its duplicated above so can consolidate it.